src/operator/tensor/init_op.h

```diff
@@ -197,14 +197,21 @@ void FillZerosRspImpl(mshadow::Stream<xpu> *s, NDArray *dst) {
   dst->set_aux_shape(rowsparse::kIdx, TShape(mshadow::Shape1(0)));
 }
 
-// Fill a CSR NDArray with zeros by updating the aux shape.
+/*! \brief Fill a CSR NDArray with zeros by updating the aux shape
+ *
+ * @tparam xpu - cpu or gpu
```

Comment:
`tparam ` -> `param`

Reply:
what?  cryptic comment :)

Reply:
tparam = template parameter. It's actually named by the IDE.

Reply:
https://www.stack.nl/~dimitri/doxygen/manual/commands.html

Reply:
Sorry about that. Learnt a new thing today 👍